plugins/libplugin.h

```diff
@@ -310,17 +310,19 @@ void rpc_scan(struct plugin *plugin,
 	      const char *guide,
 	      ...);
 
-/* Helper to scan datastore: can only be used in init callback.  *
- Returns false if field does not exist.  * path is /-separated.  Final
- arg is JSON_SCAN or JSON_SCAN_TAL.
```

Comment:
I wouldn't have thought a comment block could give me anxiety, but this one succeeded!